src/main/frontend/components/FilmShow.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+
+
+const FilmShow = props => {
+  const [film, setFilm] = useState([])

```

Comment:
If a single film will be stored here and not a list of films, I would use an empty object `{}` instead of an empty array `[]` as the initial value